tools/SetupFlow/DevHome.SetupFlow/ViewModels/SetupFlowViewModel.cs

```diff
@@ -149,8 +149,7 @@ public void OnNavigatedTo(NavigationEventArgs args)
         var parameter = args.Parameter?.ToString();
 
         if ((!string.IsNullOrEmpty(parameter)) &&
-            parameter.Contains(_creationFlowNavigationParameter, StringComparison.OrdinalIgnoreCase) &&
-            Orchestrator.CurrentSetupFlowKind != SetupFlowKind.CreateEnvironment)

```

Comment:
This change was needed to make sure the Create Environment page is refreshed every time it is navigated to from the Environments page.